diff --git a/CMakeLists.txt b/CMakeLists.txt
index 280112b..12c6ad6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,6 +17,13 @@ if (ENABLE_FORTRAN)
    enable_language(Fortran)
 endif()

+#------------------------------------------------------------------------------
+# Metall Support
+#------------------------------------------------------------------------------
+find_package(Boost 1.80)
+find_package(Threads)
+find_package(Metall)
+
 #------------------------------------------------------------------------------
 # Setup BLT
 #------------------------------------------------------------------------------
@@ -45,7 +52,7 @@ if ("${PROJECT_SOURCE_DIR}" STREQUAL "${CMAKE_SOURCE_DIR}")
     # Set some default BLT options before loading BLT only if not included in
     # another project
     if (NOT BLT_CXX_STD)
-        set(BLT_CXX_STD "c++14" CACHE STRING "")
+        set(BLT_CXX_STD "c++17" CACHE STRING "")
     endif()

     set(ENABLE_ASTYLE      OFF CACHE BOOL "")